episimlab/foi/base.py

```diff
@@ -17,8 +17,8 @@ class BaseFOI:
     risk_group = xs.global_ref('risk_group')
     vertex = xs.global_ref('vertex')
 
-    beta = xs.variable(intent='in')
-    omega = xs.variable(dims=('age_group', 'compartment'), intent='in')
+    beta = xs.variable(intent='in', global_name='beta')
+    omega = xs.variable(dims=('age_group', 'compartment'), intent='in', global_name='omega')
 
```

Comment:
Does it make sense to deprecate the submodule FOI?

Reply:
We're not deprecating FOI as a separate process. In fact, this PR introduces pytests that check whether having FOI separate from SEIR (`cy_seir_cy_foi`) vs having `seir_with_foi` produces consistent results: https://github.com/eho-tacc/episimlab/blob/0840722cd97be91747ac7b3fef18c2c9990c540d/tests/test_seir/test_seir_with_foi.py#L39-L39

I did, however, change the model returned by `episimlab.model.basic.partition` to use the new `SEIRwithFOI` process by default, in anticipation of adding `V` compartment in new process `SEIRwithFOIVacc`:

https://github.com/eho-tacc/episimlab/blob/0840722cd97be91747ac7b3fef18c2c9990c540d/episimlab/models/basic.py#L118-L118